nbgrader/exchange/exchange.py

```diff
@@ -125,3 +125,34 @@ def start(self):
         self.init_src()
         self.init_dest()
         self.copy_files()
+
+    def get_current_user_courses(self):
+        """Check if student is enrolled in course"""
+        from tornado.httpclient import HTTPClient, HTTPRequest
+
+        if os.getenv('JUPYTERHUB_API_TOKEN'):
+            api_token = os.environ['JUPYTERHUB_API_TOKEN']
+        else:
+            self.exit("JUPYTERHUB_API_TOKEN env is required to run the exchange features of nbgrader.")
+        hub_api_url = os.environ.get('JUPYTERHUB_API_URL') or 'http://127.0.0.1:8081/hub/api'
+        if os.getenv('JUPYTERHUB_USER'):
+            user = os.environ['JUPYTERHUB_USER']
+        else:
+            self.exit("JUPYTERHUB_USER env is required to run the exchange features of nbgrader.")
+        import json
```

Comment:
Probably would be better to have this import at the top of the file.

Reply:
done